docs/source/docker/advanced/code_examples/load_model_from_checkpoint.py

```diff
@@ -0,0 +1,49 @@
+from collections import OrderedDict
+import torch
+import lightly
+
+def load_ckpt(ckpt_path, model_name='resnet-18', model_width=1):
+    ckpt = torch.load(ckpt_path, map_location='cpu')
+    
+    state_dict = OrderedDict()
+    for key, value in ckpt['state_dict'].items():
+        if ('projection_head' in key) or ('backbone.7' in key):
+            continue
+        state_dict[key.replace('model.backbone.', '')] = value
+    
+    resnet = lightly.models.ResNetGenerator(name=model_name, width=model_width)
+    model = torch.nn.Sequential(
+        lightly.models.batchnorm.get_norm_layer(3, 0),
+        *list(resnet.children())[:-1],
```

Comment:
The `[:-1]` simply removes the adaptive average pooling, no?

Reply:
It removes the classification head from the resnet.